### PR TITLE
Update README documentation

### PR DESCRIPTION
**Pull request by lrapetti**
Accordingly to https://github.com/robotology/walking-controllers/issues/102

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,9 +51,10 @@ Eprint = {arXiv:1809.02167},
 * [YARP](http://www.yarp.it/): to handle the comunication with the robot;
 * [iDynTree](http://wiki.icub.org/codyco/dox/html/idyntree/html/): to handle the robot kinematics;
 * [iCubContrib](https://github.com/robotology/icub-contrib-common): to configure the modules;
+* [icub-main](https://github.com/robotology/icub-main): to handle signale processing;
```
Comment (lrapetti) [inline README.md:54]: As far as I see `icub-main` is used only for the min-jerk-trajectory and the integrator. I didn't know hot to describe it. @S-Dafarra @GiulioRomualdi feel free to suggest better description of it.

Comment (lrapetti) [reply README.md:54]: (I also noticed probably this include https://github.com/robotology/walking-controllers/blob/dcc2698e3d16adb5c78c973a27a359f0986ae535/src/SimplifiedModelControllers/include/WalkingControllers/SimplifiedModelControllers/ZMPController.h#L19 is not required

Comment (S-Dafarra) [inline README.md:54]:
```suggestion
* [icub-main](https://github.com/robotology/icub-main): to smooth and integrate signals;
```

Comment (S-Dafarra) [reply README.md:54]: Feel free to remove it and add that modification to this PR

**Comment by lrapetti** [comment]
Hi @S-Dafarra the comment should have been addressed, please let me know if any additional change is required

**Review by S-Dafarra — APPROVED** [review]

**Pull request merged**